### PR TITLE
Fix even more lint warnings

### PR DESCRIPTION
**Pull request by Kubuxu**
Down to 121 lint warnings

### CODE REVIEW

File: build/version.go [hunk 1]
```diff
@@ -55,7 +55,7 @@ func (ve Version) EqMajorMinor(v2 Version) bool {
 // APIVersion is a semver version of the rpc api exposed
 var APIVersion Version = newVer(0, 3, 0)
 
-//nolint:varcheck
+//nolint:varcheck,deadcode
```
Comment (whyrusleeping) [inline build/version.go:58]: if this is dead code shouldnt we just remove it?

Comment (Kubuxu) [reply build/version.go:58]: It is about maskMajor and maskMinor (or two other), and I feel like it is fine to have them for the sake of completeness.

**Comment by Kubuxu** [comment]
These tests passed locally. Rerunning on CircleCI

**Comment by Kubuxu** [comment]
Two lint warnings that are reported are expected. They need a deeper fixing.

**Pull request merged**